### ISSUE TITLE
Question: how to use multiple variables in tags?

### ISSUE THREAD

**Issue by zblesk** [joplin/plugin-templates#55]
I have something like this:

```
---
book: text
genre: text
status: dropdown(, finished, unfinished)
template_tags: books, {{genre}},{{status}}

---
```

That is, I want to have one tag fixed, one as free text, and one from a dropdown. This works as long as I enter all of them; but if I leave for example the `genre` empty, just the first tag gets added. 

Maybe it is because it ends up as `books, , finished` and the comma throws it off? 
I have tried using {{#if}} in that line, but got nowhere. Is this possible? How?

**Comment by nishantwrp** [comment]
Will soon release a fix for this.

**Comment by nishantwrp** [comment]
Released in `v2.2.1`.
